src/api/app/queries/requests_finder.rb

```diff
@@ -0,0 +1,13 @@
+class RequestsFinder
```

Comment:
Is there a reason you didn't add this to `OpenRequestsFinder` but created your own query object?

Reply:
@hennevogel I'm addressing it with https://github.com/openSUSE/open-build-service/pull/13042/commits/651f9b9215f172d372a267c4d8f3c831dd13626e , what do you think?